config/routes.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-Spree::Core::Engine.routes.draw do
+SolidusGraphqlApi::Engine.routes.draw do
```

Comment:
Not sure if this is the right approach within the Solidus/Spree project, but it's what the Rails docs recommend. I can change it back if you'd like.